### DISCUSSION TITLE
How can I start RustDesk with an URL on Mac

### DISCUSSION THREAD

**Discussion by nico-hl** [rustdesk/rustdesk#4590] [General]
Hey,
anyone has an idea how can I start RustDesk with an URL on Mac ? 
I wish to start RustDesk with ID and password input so RustDesk connects directly to the client. I hope you know what I mean :D 
url://rustdesk.ID1234567.PWxxxxxxx.com and now RustDesk open with connection to ID 1234567

Thanks
Nico

**Comment by djordans** [comment]
Hello,

`rustdesk://connection/new/12345?password=1234`

**Reply by nico-hl** [reply]
Hi, 
this doesn't work on Mac or is there any hint that I don't know :D

**Comment by nico-hl** [comment]
So now it works. 👍  I had an old version of RustDesk installed
